govtool/packages/wallet-connector/components/ChooseWalletModal.tsx

```diff
@@ -0,0 +1,140 @@
+import { useMemo } from "react";
+import { Box, Link, Modal, Typography } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+
+import { WalletButton } from "./WalletButton";
+import { ChooseWalletModalProps, WalletInfo } from "./types";
+
+export const ChooseWalletModal = ({
+  handleClose,
+  isOpen,
+  isWalletEnableLoading,
+  onClickShowSupportedWallets,
+  onClickWalletButton,
+}: ChooseWalletModalProps) => {
+  const walletOptions = useMemo(() => {
+    if (!window.cardano) return [];
+    const keys = Object.keys(window.cardano);
+    let resultWallets: WalletInfo[] = [];
```

Comment:
why not just map `window.cardano` here?

Reply:
it could be reduce with map, but this function already exist in govtool then I've reused it

Reply:
I know, but still we can reduce the declarations here easily leaving the code better than it used to be 🫡